beacon_node/client/src/builder.rs

```diff
@@ -755,6 +774,27 @@ where
         }
 
         if let Some(beacon_chain) = self.beacon_chain.as_ref() {
+            if let Some(network_globals) = &self.network_globals {
+                let beacon_processor_context = runtime_context.service_context("bproc".into());
+                BeaconProcessor {
```

Comment:
The `BeaconProcessor` is now instantiated and started here in the `ClientBuilder` rather than in the `network` crate.